test/testsuite/lantest_io_monitor.h

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2025 Netatalk Project
```

Comment:
Would you feel comfortable putting yourself as the copyright owner here?

Even if the code would be considered derivative of existing netatalk code, I think it's preferable to have an individual's name to indicate authorship.

What do you think?

Reply:
Lol, sure 😉

Reply:
What about lantest.c? I have updated more than 50% of the code now?

Reply:
lol, same 😉 Ok I'll add one and include them

Reply:
Done